spec/models/glossary_spec.rb

```diff
@@ -0,0 +1,92 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Glossary, type: :mode do
+  let(:glossaryMock) { build(:glossary) }
```

Comment:
What is this filthy camelcasing im my beautiful Ruby code? 😂

Reply:
Forgive me, oh merciful Ruby mentor! I have JavaScript sinned 😂